ocean_lib/data_provider/data_service_provider.py

```diff
@@ -141,13 +141,13 @@ def initialize(
         service_endpoint += f"?documentId={did}"
         payload = {"serviceId": service_id, "consumerAddress": consumer_address}
 
-        if compute_environment:
+        if compute_environment is not None:
```

Comment:
I'm curiuous:
What's the difference between `bool(compute_environment)` and `bool(compute_environment is not None)` ?